@rocketseat/gatsby-theme-docs-core/gatsby-node.js

```diff
@@ -22,7 +22,12 @@ exports.createPages = (
   return graphql(
     `
       {
-        files: allFile(filter: { extension: { in: ["md", "mdx"] } }) {
+        files: allFile(
+          filter: {
+            extension: { in: ["md", "mdx"] }
+            sourceInstanceName: { eq: "docs" }
```

Comment:
Should this not be `docsPath`?

Reply:
I don't think so, the name of the gatsby-source-filesystem instance that pulls from `docsPath` in gatsby-config.js is `docs` so it's filtering for only those files.